### ISSUE TITLE
triplot() does not work in my case..

### ISSUE THREAD

**Issue by Nehagupta90** [ModelOriented/triplot#23]
Hello everyone

I am using triplot library for feature selection but it give me error.. I simply train a model using mlr3 and create an explainer and then the following :

library(triplot)
model_triplot(explainer)

The error message is:

Error in hclust(as.dist(1 - abs(cor(data, method = cor_method))), method = clust_method) : 
  NA/NaN/Inf in foreign function call (arg 10)
In addition: Warning message:
In cor(data, method = cor_method) : the standard deviation is zero

**Comment by hbaniecki** [comment]
@Nehagupta90 there is an `explain_mlr3()` function in the [`DALEXtra`](https://github.com/ModelOriented/DALEXtra) package. Please try it out and (if it fails) provide a minimal reproducible example to debug this issue.

**Comment by Nehagupta90** [comment]
Thank you for your email.

I have already used explain_mlr3.

data = readARFF("china.arff")

index= sample(1:nrow(data), 0.7*nrow(data))
train= data[index,]
test= data[-index,]

task = TaskRegr$new("data", backend = train, target = "Effort")
learner= lrn("regr.randomForest")
model= learner$train(task )

explainer = explain_mlr3(model,
                         data = test[,-19],
                         y = test$Effort,
                         label="RF")

library(triplot)
model_triplot(explainer)

**Comment by Nehagupta90** [comment]
Hi, I am still waiting for the problem to be resolved.. Why I am getting
the above error?

Warm regards

**Comment by p-schaefer** [comment]
@Nehagupta90 Did you ever resolve this problem? I'm having the same issue.

Edit: I think this has to do with NA's in the data. Being able to pass arguments to ```cor(use =x)``` [here](https://github.com/ModelOriented/triplot/blob/a9721315b13228ecf6998125c3c36be188d9355b/R/hierarchical_importance.R#L83) might help.

**Comment by Nehagupta90** [comment]
Yes but with some models it works and with some it gives problems.. I am
not sure why we are not getting response from experts here.
